index.js

```diff
@@ -16,9 +25,9 @@ function setup() {
     dots.push(new Dot());
     dots[i].pos.x = Math.random() * width;
     dots[i].pos.y = Math.random() * height;
-    dots[i].rad = 15 + Math.random() * 10;
-    dots[i].vel.x = Math.random() * 5;
-    dots[i].vel.y = Math.random() * 5;
+    dots[i].rad = 15;
+    dots[i].vel.x = (Math.random() - 0.5);
```

Comment:
Whooooa, I've never thought to do this. I've always done `(Math.random() > 0.5 ? 1 : -1) * expression`